src/core/agent/services/identifierService.ts

```diff
@@ -283,11 +283,40 @@ class IdentifierService extends AgentService {
     if (unSyncedData.length) {
       //sync the storage with the signify data
       for (const identifier of unSyncedData) {
-        await this.identifierStorage.createIdentifierMetadataRecord({
-          id: identifier.prefix,
-          displayName: identifier.prefix, //same as the id at the moment
-          theme: 0,
-        });
+        if(identifier.group){
+          const groupId = identifier.group.mhab.name.split(":")[1];
+          const groupMetadata = {
```

Comment:
`groupMetadata` is for the mhab identifier, not ghab, right? i.e. the single sig we originally create that is in the `smids` array.

Reply:
`group` exists from signify only for the multi-sig. I think you're mixing these up, unless my memory is off